dev/breeze/src/airflow_breeze/utils/run_utils.py

```diff
@@ -470,7 +471,15 @@ def run_compile_www_assets(
         f"{WWW_ASSET_OUT_DEV_MODE_FILE if dev else WWW_ASSET_OUT_FILE}\n"
     )
     if run_in_background:
-        thread = Thread(daemon=True, target=_run_compile_internally, args=(command_to_execute, dev))
-        thread.start()
+        pid = os.fork()
+        if pid:
+            # Parent process - send signal to process group of the child process
+            atexit.register(os.killpg, pid, signal.SIGTERM)
+        else:
+            # Check if we are not a group leader already (We should not be)
+            if os.getpid() != os.getsid(0):
+                # and create a new process group where we aer the leader
```

Comment:
```suggestion
                # and create a new process group where we are the leader
```

Reply:
Yeah. I am slightly dyslexsic :) that's a fact :). That's why I love spellchecking pre-commits -but they don't work on such short words :).